src/lib/containers/widgets/facet-nav/FacetNavPanel.tsx

```diff
@@ -351,7 +351,7 @@ const FacetNavPanel: React.FunctionComponent<FacetNavPanelProps> = ({
       )
       setPlotData(plotData)
     }
-  }, [facetToPlot, data])
+  }, [facetToPlot, data, index])
```

Comment:
Added variable to monitor since it was referenced in the useEffect hook (removes a react build warning).